2019/10/21 19:03:21 [error] 6#6: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.4:9000", host: "localhost:8083"
2019/10/21 19:03:21 [error] 6#6: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.20.0.4:9000", host: "localhost:8083", referrer: "http://localhost:8083/"
2019/10/21 19:34:46 [error] 6#6: *5 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.4:9000", host: "localhost:8083"
2019/10/21 20:22:13 [error] 6#6: *9 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET /docs HTTP/1.1", upstream: "fastcgi://172.20.0.4:9000", host: "localhost:8083"
2019/11/02 19:23:50 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "localhost:8083"
2019/11/02 19:42:47 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET /devices HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "localhost:8083"
2019/11/02 19:43:27 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading upstream, client: 172.20.0.1, server: localhost, request: "GET /devices HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "localhost:8083"
2019/11/02 19:44:32 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET /devices HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "localhost:8083"
2019/11/02 19:45:12 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading upstream, client: 172.20.0.1, server: localhost, request: "GET /devices HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "localhost:8083"
2019/11/02 19:48:27 [error] 7#7: *6 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET /devices HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "localhost:8083"
2019/11/02 19:49:07 [error] 7#7: *6 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading upstream, client: 172.20.0.1, server: localhost, request: "GET /devices HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "localhost:8083"
2019/11/02 19:51:01 [error] 6#6: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET /devices HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "localhost:8083"
2019/11/02 19:51:01 [error] 6#6: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET /devices/ HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "localhost:8083"
2019/11/02 19:52:29 [error] 6#6: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET /devices HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "localhost:8083"
2019/11/02 19:52:29 [error] 6#6: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET /devices/ HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "localhost:8083"
2019/11/02 19:53:11 [error] 6#6: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET /devices HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "localhost:8083"
2019/11/02 19:53:11 [error] 6#6: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET /devices/ HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "localhost:8083"
2019/11/02 19:53:11 [error] 6#6: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading upstream, client: 172.20.0.1, server: localhost, request: "GET /devices/ HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "localhost:8083"
2019/11/02 19:54:00 [error] 6#6: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET /devices HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "localhost:8083"
2019/11/02 19:54:00 [error] 6#6: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'.PHP message: Xdebug could not open the remote debug file '/app/var/log/xdebug.log'" while reading upstream, client: 172.20.0.1, server: localhost, request: "GET /devices HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "localhost:8083"
2019/11/02 20:53:03 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.4:9000", host: "localhost:8083"
2019/11/02 20:53:03 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.20.0.4:9000", host: "localhost:8083", referrer: "http://localhost:8083/"
2019/11/02 20:53:14 [error] 6#6: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: localhost, request: "GET /devices HTTP/1.1", upstream: "fastcgi://172.20.0.4:9000", host: "localhost:8083"
2019/11/02 20:59:29 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: localhost, request: "GET /devices HTTP/1.1", upstream: "fastcgi://172.20.0.4:9000", host: "localhost:8083"
2019/11/02 22:55:30 [error] 8#8: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET /devices HTTP/1.1", upstream: "fastcgi://172.22.0.4:9000", host: "localhost:8083"
2019/11/02 22:55:32 [error] 8#8: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET /devices HTTP/1.1", upstream: "fastcgi://172.22.0.4:9000", host: "localhost:8083"
